apps/chrome-devtools/src/app-devtools/theming-panel/theming-panel-pres.template.html

```diff
@@ -1,61 +1,117 @@
+                  <i class="form-text text-warning icon-undo position-absolute end-0 me-10" ngbTooltip="Reset all colors of {{group.name}}" (click)="onPaletteReset(group, $event)"></i>
+                }
+              } @else {
+                <label class="form-label text-nowrap fw-semibold my-0">{{ group.name }}</label>
               }
-              <ng-template #resultTemplate let-result="result" let-term="term">
-                <div class="d-flex flex-column gap-2">
-                  <ngb-highlight [result]="result" [term]="term"></ngb-highlight>
-                  @if (result | variableName; as varName) {
-                    @if (variablesMap()[varName]; as variable) {
-                      <div class="d-flex gap-3 fs-6 text-secondary align-items-center">
-                        @if (variable.type === 'color') {
-                          <div class="border border-secondary rounded-circle" [style.background]="resolvedVariables()[variable.name] | color" [style.width]="'15px'" [style.height]="'15px'"></div>
-                        }
-                        @if (variable.description; as description) {
-                          <ngb-highlight [result]="description" [term]="term"></ngb-highlight>
-                        } @else {
-                          <ngb-highlight [result]="variable.runtimeValue ?? variable.defaultValue" [term]="term"></ngb-highlight>
-                        }
-                      </div>
+            </button>
+          </h3>
+          <div ngbAccordionCollapse>
+            <div ngbAccordionBody>
+              <ng-template>
+                <ul class="d-grid w-100 overflow-hidden p-0 border-top border-start border-light" [style.grid-template-columns]="group.isPalette ? null : 'repeat(auto-fill, minmax(300px, 1fr))'">
+                  @for (variable of group.variables; track variable.name) {
+                    @if (group.isPalette) {
+                      @if (resolvedVariables()[variable.name] | hexColor; as resolvedColor) {
+                        <li class="d-flex px-4 gap-4" [style.background]="resolvedColor" [style.color]="resolvedColor | contrast">
+                          <label class="form-label text-nowrap fw-semibold my-auto" [for]="variable.name">
+                            {{ variable | variableLabel }}
+                          </label>
+                          <span class="my-auto">{{ resolvedColor }}</span>
+                          <span class="ms-auto my-auto">
+                            <span ngbTooltip="Accessibility score for small text" [style.fontSize]="'12px'" class="me-2">{{ resolvedColor | accessibilityConstrastScore : (resolvedColor | contrast) : 'small' }}</span>
+                            <span ngbTooltip="Accessibility score for large text" [style.fontSize]="'24px'">{{ resolvedColor | accessibilityConstrastScore : (resolvedColor | contrast) : 'large' }}</span>
```

Comment:
why use style.fontSize and use a hard coded value?

Reply:
hardcoded value because for accessibility large text start at 24px and for small text I picked 12px to let the user see a big difference between large and small without having it too small to be read

Reply:
for `[style.fontSize]="'24px'"` I found it clearer than `style="{font-size: 24px'}` and Angular provide autocomplete for `[style.<...>]`